patches/@protobufjs+inquire+1.1.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@protobufjs/inquire/index.js b/node_modules/@protobufjs/inquire/index.js
+index 33778b5..3115fa7 100644
+--- a/node_modules/@protobufjs/inquire/index.js
++++ b/node_modules/@protobufjs/inquire/index.js
+@@ -9,7 +9,7 @@ module.exports = inquire;
+  */
+ function inquire(moduleName) {
+     try {
+-        var mod = eval("quire".replace(/^/,"re"))(moduleName); // eslint-disable-line no-eval
++        var mod = require(moduleName); // eslint-disable-line no-eval
```

Comment:
is there a tracking issue in protobufjs to remove eval? if not, we should create one. If so, we should link to it.

Reply:
100% agree with your not wanting to deal with patches, i just wanted to make sure there was an issue open with them regarding the use of eval.

Unfortunately, I doubt that issue will ever be resolved. protobuf.js repo has a large number of open issues and PRs.